main.py

```diff
@@ -153,9 +167,9 @@ def get_topology(self):
     def restore_network_status(self):
         """Restore the network administrative status saved in StoreHouse."""
         try:
-            switches_status, interfaces_status = self._load_network_status()
-            self._restore_status(switches_status, interfaces_status)
-        except (KeyError, FileNotFoundError) as exc:
+            self._load_network_status()
+            self._restore_status()
+        except (TypeError, FileNotFoundError) as exc:
```

Comment:
```suggestion
            self._load_network_status()
            self._restore_status()
        except (KeyError, FileNotFoundError) as exc:
```
You changed the behavior of `_load_network_status`. Now, if it fails, it will raise `KeyError`. As it was before, it could raise both `KeyError` or `TypeError`.

Reply:
Thanks, I made the requested change. Returning the handler to `KeyError`.